Makefile

```diff
@@ -30,7 +30,7 @@ EXCLUDED_MODULES=\
 	$(DJANGO_PROJECT_ROOT)/pcari/test*\
 	$(DJANGO_PROJECT_ROOT)/pcari/urls.py
 
-LOCALES=tl
+GET_LOCALE_CMD=./manage.py shell -c 'from django.conf import settings; print(" ".join(code for code, _ in settings.LANGUAGES if not code.startswith("en")))'
```

Comment:
Is there a cleaner way to do this? Scraping from shell seems a bit jank.

Reply:
I agree, but this way, you don't have to update the list of locales in two places.